wscript

```diff
@@ -548,7 +548,7 @@ def configure(cfg):
 
     # require python 3.8.x or later
     cfg.load('python')
-    cfg.check_python_version(minver=(3,6,9))
+    cfg.check_python_version(minver=(3,8,0))
```

Comment:
maybe if version_is_3.6, print big warning, sleep 5 seconds to user can see it

Reply:
< 3.8?